readme.md

```diff
@@ -68,14 +68,13 @@ Below will render an image:
 
 ## API
 
-This package exports no identifiers.
+This package exports `defaultImageExtensions`.
 The default export is `remarkImages`.
 
-### `unified().use(remarkImages)`
+### `unified().use(remarkImages[, options])`
 
 Add an improved image syntax.
-Transform URLs in text that reference images (`png`, `svg`, `jpg`, `jpeg`, or
-`gif`) to images.
+Transform URLs in text that reference images (`png`, `svg`, `jpg`, `jpeg`, `gif`, `webp`, or `avif`) to images.
```

Comment:
I think the `defaultImageExtensions` export should also be documented, then this list can be removed.

Reply:
added